client/src/components/editor/EditorSideBarData.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {BsFillGridFill, BsFillHouseDoorFill, BsBookHalf, 
```

Comment:
Since you are updating the code, can you remove the unused imports `BsBookHalf` as well? So that we wont need to patch it agian when merging into main later. If posibble, can you also remove the `Image` import in `Header.js`, I forgot to remove it in last PR XD

Reply:
OK so what icon should I have in place of books half (books half was there for the publications page)

Reply:
I think your publication page is using BsFillGridFill currently? You might need to change it and remove the BsFillGridFill from import.